### ISSUE TITLE
Question about classification inference input

### ISSUE THREAD

**Issue by ChaozhongLiu** [TencentAILabHealthcare/DNAGPT#5]
Hi there,

Thanks for the great work!
I'm learning the token language you have designed, and got a question regarding the input sequence in [this](https://github.com/TencentAILabHealthcare/DNAGPT/blob/main/scripts/classification.sh#L5) command example:

Why in the end of the input sequence, we have ``<=><R>`` instead of ``<=><B>`` to represent "this is a binary classification task" ?
Is it because the fine-tuning is designed like so?
If so, what about the pre-training input sequence to represent the ``Sequence order prediction`` label? ``<=><R>`` or ``<=><B>``?

Thanks in advance!
